constellation_base/signals.py

```diff
@@ -0,0 +1,15 @@
+from django.dispatch import receiver
+from django.contrib.auth.signals import user_logged_in
+
+from .models import LocalGroupACL
+
+
+@receiver(user_logged_in)
+def addSupplementalGroups(sender, user, request, **kwargs):
+    """Apply additional groups on login"""
+    for group in list(LocalGroupACL.objects.all()):
+        supGroup = group.group
+        if user.username in group.members:
```

Comment:
Consider the group `admin`, containing a single user `admin` or `maldridge`...What happens when someone creates a user with the name `a`?